RuntimeUnityEditor/Windows/ObjectTree/ObjectTreeViewer.cs

```diff
@@ -23,10 +23,11 @@ public sealed class ObjectTreeViewer : Window<ObjectTreeViewer>
         private Vector2 _propertiesScrollPosition;
         private Vector2 _treeScrollPosition;
         private float _objectTreeHeight;
-        private int _singleObjectTreeItemHeight;
+        private float _singleObjectTreeItemHeight;
+        private float _singleObjectTreeItemMargin;
 
         private bool _scrollTreeToSelected;
-        private int _scrollTarget;
+        private float _scrollTarget = -1f;
```

Comment:
_scrollTarget of 0 was causing it to not get caught in Update()'s check for `if (_scrollTreeToSelected && _scrollTarget > 0)` if scrollTarget was 0. Set to -1 makes it clear when not in use and 0 becomes valid when selecting first item.